tests/bluetooth/bsim_bt/compile.sh

```diff
@@ -20,29 +20,4 @@ mkdir -p ${WORK_DIR}
 
 source ${ZEPHYR_BASE}/tests/bluetooth/bsim_bt/compile.source
 
-app=tests/bluetooth/bsim_bt/bsim_test_app conf_file=prj_split.conf \
```

Comment:
I don't think this should be erased - for testing only audio we can comment out the rest if need be.

Reply:
Yes, I've pushed it by mistake

Reply:
done